ostelco-ios-client/ViewControllers/Country/LocationAccessDeniedViewController.swift

```diff
@@ -29,10 +29,16 @@ class LocationAccessDeniedViewController: UIViewController {
             .debug("didChangeAuthorization")
             .filter({_,status in
                 switch status {
-                case .restricted, .denied:
+                case .restricted,
+                     .denied:
                     return false
-                case .authorizedAlways, .authorizedWhenInUse, .notDetermined:
+                case .authorizedAlways,
+                     .authorizedWhenInUse,
+                     .notDetermined:
                     return true
+                @unknown default:
```

Comment:
This is a new thing in Swift 5 that essentially allows you to add a default case for enums you don't control and which might in the future have other values. 

If you're extremely bored [please enjoy the Swift Evolution proposal](https://github.com/apple/swift-evolution/blob/master/proposals/0192-non-exhaustive-enums.md) which led to this change.